src/expr/scalar/func.rs

```diff
@@ -1994,7 +2014,8 @@ impl fmt::Display for BinaryFunc {
             BinaryFunc::MatchLikePattern => f.write_str("like"),
             BinaryFunc::ToCharTimestamp => f.write_str("tocharts"),
             BinaryFunc::ToCharTimestampTz => f.write_str("tochartstz"),
-            BinaryFunc::DateTrunc => f.write_str("date_trunc"),
+            BinaryFunc::DateTruncTimestamp => f.write_str("date_trunc"),
+            BinaryFunc::DateTruncTimestampTz => f.write_str("date_trunc"),
```

Comment:
These should probably follow the convention of `tochar` and be `datetruncts` and `datetrunctstz`.